cloud/amazon/ec2_vpc_peering.py

```diff
@@ -0,0 +1,611 @@
+#!/usr/bin/env python
+"""
+Ansible module to manage VPC peering connections
+"""
+
+DOCUMENTATION = '''
+---
+module: ec2_vpc_peering
+short_description: manage AWS VPC Peering Connections
+description:
+    - Create, list or reject/delete AWS VPC peering connections.
+    - This module has a dependency on python-boto
+version_added: "2.0"
+author: Herby Gillot <herby.gillot@gmail.com>
```

Comment:
Let's make sure that both your Github ID and Herby's Github ID are included here.